### PR TITLE
feat(shared-research-output): show subtype everywhere types appear

### PR DESCRIPTION
**Pull request by jeysal**

### CODE REVIEW

File: packages/react-components/src/atoms/TagLabel.tsx [hunk 1]
```diff
@@ -15,22 +15,23 @@ const styles = css({
...
 
+  backgroundColor: paper.rgb,
+  color: lead.rgb,
+
   borderStyle: 'solid',
   borderWidth: `${borderWidth}px`,
   borderColor: steel.rgb,
   borderRadius: `${6 / perRem}em`,
-
-  fontSize: '0.8em',
```
Comment (jeysal) [inline packages/react-components/src/atoms/TagLabel.tsx:23]: was causing problems because it affected what `em` means in other values

Comment (jeysal) [reply packages/react-components/src/atoms/TagLabel.tsx:23]: replaced by a `small` tag

File: packages/react-components/src/organisms/SharedResearchCard.tsx [hunk 2]
```diff
@@ -1,70 +1,37 @@
...
-        >
-          <Headline2 styleAsHeading={4}>{title}</Headline2>
-        </Anchor>
-        <TeamsList inline max={3} teams={teams} />
-      </div>
-      <Caption accent={'lead'} asParagraph>
-        Date Added: {formatDate(new Date(addedDate || created))}
-      </Caption>
-    </div>
+    <SharedResearchMetadata {...props} />
```
Comment (jeysal) [inline packages/react-components/src/organisms/SharedResearchCard.tsx:25]: Soooo clean now, literally just a card with other things in it 😍

File: packages/react-components/src/templates/__tests__/TeamProfileOutputs.test.tsx [hunk 3]
```diff
@@ -34,8 +33,8 @@ it('renders output cards', () => {
   expect(links).toHaveLength(2);
   const [titleLink, teamLink] = links;
 
-  expect(titleLink.textContent).toMatchInlineSnapshot(`"Title"`);
-  expect(teamLink.textContent).toMatchInlineSnapshot(`"Team Unknown"`);
+  expect(titleLink).toHaveTextContent('Title');
+  expect(teamLink).toHaveTextContent(/Unknown/);
```
Comment (kmaid) [inline packages/react-components/src/templates/__tests__/TeamProfileOutputs.test.tsx:37]: not case insensitive

**Review by kmaid — APPROVED** [review]

File: packages/react-components/src/templates/__tests__/TeamProfileOutputs.test.tsx (hunk 3 above)
Comment (jeysal) [reply packages/react-components/src/templates/__tests__/TeamProfileOutputs.test.tsx:37]: Yeah, this string comes from the input data not some copy

**Pull request merged**